<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageDialog</class>
 <widget class="QDialog" name="MessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>1026</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_no">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>137</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_no_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>650</y>
     <width>71</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMessage">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>970</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeMessage">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>970</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="receiverMessage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>970</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_no_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>920</y>
     <width>137</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Receiver</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputMessageTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_no_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>61</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputMessage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>700</y>
     <width>651</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="refereshMessage">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>새로고침</string>
   </property>
  </widget>
  <widget class="QListView" name="outputMessage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>651</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
